pyband/messages/cosmos/bank/v1beta1/msgs.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+
+from ....common import MessageWrapper
+from .....proto.cosmos.bank.v1beta1 import MsgMultiSend as MsgMultiSendProto
+from .....proto.cosmos.bank.v1beta1 import MsgSend as MsgSendProto
+
+try:
+    from .....proto.cosmos.base import v1beta1 as __base_v1_beta1__
+    from .....proto.cosmos.base.query import v1beta1 as __base_query_v1_beta1__
+except ImportError as ie:
+    raise ie
```

Comment:
Why need to have try/except if you still raise exception

Reply:
changed to assert